### PR TITLE
fix(docs): update help docs for zarf connect to add clarity

### LINKED ISSUE #2661: Doc update for zarf connect
### Environment
Device and OS: Apple M3 Max
App version: 0.33.1
Kubernetes distro being used: K3s
Other:

### Steps to reproduce
1. `zarf init`
2. `zarf package deploy oci://defenseunicorns/dos-games:1.0.0-$(uname -m) --key=https://zarf.dev/cosign.pub`
3. `zarf connect doom --local-port 8080`

### Expected result
Doom is reachable at `http://127.0.0.1:8080`

### Actual Result
Doom is reachable at a randomly selected port

### Severity/Priority
Low

### Additional Context
This isn't so much of a bug as it is ambiguous help output for the `zarf connect` subcommand.  If `--name` option is blank, then  `--local-port`, `--remote-port`, `--type`, and `--namespace` are all ignored and the service is found using labels.  This has lead to some confusion for users who expect to be able to specify a local port to forward on using `--local-port`.

### PR DESCRIPTION
**Pull request by chaospuppy**
## Description
Adds some text to the help output for `zarf connect` to make it clearer that most of the flags don't do anything if `--name` is unset.  This could be taken a step further if desired to rework how the tunnel is constructed to allow for the specification of one or more of `--remote-port`, `--local-port`, `--type`, and `--namespace` without specifying `--name`.

## Related Issue
Fixes #2661

## Checklist before merging

- [ ] Test, docs, adr added or updated as needed
- [ ] [Contributor Guide Steps](https://github.com/defenseunicorns/zarf/blob/main/.github/CONTRIBUTING.md#developer-workflow) followed

### CODE REVIEW

File: site/src/content/docs/commands/zarf_connect.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ zarf connect { REGISTRY | LOGGING | GIT | connect-name } [flags]
 ```
       --cli-only           Disable browser auto-open
   -h, --help               help for connect
-      --local-port int     (Optional, autogenerated if not provided) Specify the local port to bind to.  E.g. local-port=42000
-      --name string        Specify the resource name.  E.g. name=unicorns or name=unicorn-pod-7448499f4d-b5bk6
-      --namespace string   Specify the namespace.  E.g. namespace=default (default "zarf")
-      --remote-port int    Specify the remote port of the resource to bind to.  E.g. remote-port=8080
-      --type string        Specify the resource type.  E.g. type=svc or type=pod (default "svc")
+      --local-port int     (Optional, autogenerated if not provided) Specify the local port to bind to.  E.g. local-port=42000. Ignored if --name is unset
+      --name string        Specify the resource name.  E.g. name=unicorns or name=unicorn-pod-7448499f4d-b5bk6. Ignored if --name is unset
```
Comment (lucasrod16) [inline site/src/content/docs/commands/zarf_connect.md:33]: Probably an oversight, but the `Ignored if --name is unset` is part of the `--name` flag description but should not be

File: src/config/lang/english.go [hunk 2]
```diff
@@ -77,11 +77,11 @@ const (
 	// zarf connect list
 	CmdConnectListShort = "Lists all available connection shortcuts"
 
-	CmdConnectFlagName       = "Specify the resource name.  E.g. name=unicorns or name=unicorn-pod-7448499f4d-b5bk6"
-	CmdConnectFlagNamespace  = "Specify the namespace.  E.g. namespace=default"
-	CmdConnectFlagType       = "Specify the resource type.  E.g. type=svc or type=pod"
-	CmdConnectFlagLocalPort  = "(Optional, autogenerated if not provided) Specify the local port to bind to.  E.g. local-port=42000"
-	CmdConnectFlagRemotePort = "Specify the remote port of the resource to bind to.  E.g. remote-port=8080"
+	CmdConnectFlagName       = "Specify the resource name.  E.g. name=unicorns or name=unicorn-pod-7448499f4d-b5bk6. Ignored if --name is unset"
```
Comment (lucasrod16) [inline src/config/lang/english.go:80]: Probably an oversight, but `Ignored if --name is unset` is part of the `--name` flag description but shouldn't be

**Review by lucasrod16 — CHANGES_REQUESTED** [review]
@chaospuppy thanks for the PR! This is actually a bug that should be fixed in the code, but I think it's good to update the documentation to reflect current behavior. `--local-port` should work without specifying all of the other flags

**Review by phillebaba — CHANGES_REQUESTED** [review]
Could you please add punctuation to the end.

File: src/config/lang/english.go (hunk 2 above)
Comment (chaospuppy) [reply src/config/lang/english.go:80]: My bad, will fix that

**Review by phillebaba — APPROVED** [review]

**Pull request merged**